dbms/src/Interpreters/ExpressionJIT.cpp

```diff
@@ -555,7 +565,7 @@ static bool isCompilable(llvm::IRBuilderBase & builder, const IFunctionBase & fu
     return function.isCompilable();
 }
 
-void compileFunctions(ExpressionActions::Actions & actions, const Names & output_columns, const Block & sample_block)
+void compileFunctions(ExpressionActions::Actions & actions, const Names & output_columns, const Block & sample_block, std::shared_ptr<CompiledExpressionCache> compilation_cache)
```

Comment:
I realized that I also want total time spent on compiling in ProfileEvents.

`CompileExpressionsMicroseconds`